AssistantV1.playground/Pages/Workspaces.xcplaygroundpage/Contents.swift

```diff
@@ -24,13 +24,13 @@ assistant.listWorkspaces() {
 
 //:### Create workspace
 
-var workspaceID = "{workspace_id}"
+var workspaceID = ""
```

Comment:
The API docs commonly use templated initial values like "{workspace_id}" to indicate that the user must supply this value. I am just continuing that convention in the playground.  Is there a compelling reason to change it?